### PR TITLE
Add canImpersonate to the rights endpoint

### PR DESCRIPTION
**Pull request by pfeyssaguet**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,7 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.85.0] - 2024-01-05
 
 ### Added
-- [license] Add Merket Segment to License 
+- [license] Add Merket Segment to License
```
Comment (BlondJP) [inline CHANGELOG.md:40]: *Market ?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.90.0",
+  "version": "3.91.0",
```
Comment (BlondJP) [inline package.json:7]: This change should be in a different commit named like this:
**chore(package): version 3.91.0**

You can do a git log if you want to check

**Review by BlondJP — APPROVED** [review]

**Review by fdianoux — APPROVED** [review]
